helm-charts/FATE/values.yaml

```diff
@@ -51,6 +51,15 @@ modules:
     nodeSelector:
     tolerations:
     affinity:
+    polling:
+      enabled: false
+      type: client
+      server:
+        ip: 192.168.9.1
+        port: 9370
+    # clientList:
```

Comment:
remove this comment if it is not required, otherwise open it and give them default values.